whois-api/src/main/java/net/ripe/db/whois/api/rest/AbuseContactService.java

```diff
@@ -67,7 +67,7 @@ public Response lookup(
 
         final List<AbuseResources> abuseResources = Lists.newArrayList();
 
-        final int contextId = System.identityHashCode(Thread.currentThread());
+        final String contextId = String.valueOf(System.identityHashCode(Thread.currentThread()));
```

Comment:
Why is contextId changed from an int to a String? This causes a UUID string to be written into the query log.

Reply:
This was part of the _to revisit_ list I had. 

This change was because in Netty 3 [Channel.getId()](https://netty.io/3.9/api/org/jboss/netty/channel/Channel.html#getId()) returns an `int` but in Netty 4 it is replaced with [Channel.Id()](https://netty.io/4.1/api/io/netty/channel/Channel.html#id--) which returns a [ChannelId](https://netty.io/4.1/api/io/netty/channel/ChannelId.html) which is not an `int`. 

`ChannelId` has two public instance method `asShortText ` and `asLongText` which all returns string...hence the change from `int` to `string`.

What is the impact of this change to the query log? Will it break existing script/configuration? Another option which keeps `int` I could go with is to use `ChannelId.hashCode()` instead of `asShortText`/`asLongText`

Reply:
Updated to use `Channel.Id().hashCode()`. The numeric value it produces in the log is more or less the same format/size as we have before